x-pack/plugins/security_solution/server/lib/detection_engine/routes/utils.ts

```diff
@@ -296,37 +296,45 @@ export const convertToSnakeCase = <T extends Record<string, unknown>>(
 
 export const mergeStatuses = (
   id: string,
-  failures: Array<SavedObjectsFindResult<IRuleStatusAttributes>>,
+  failures: Array<SavedObjectsFindResult<IRuleStatusSOAttributes>>,
   acc: RuleStatusResponse
-) => {
+): RuleStatusResponse => {
   if (failures.length === 0) {
     return {
       ...acc,
     };
   }
-  const convertedCurrentStatus = convertToSnakeCase<IRuleStatusAttributes>(failures[0].attributes);
+  const convertedCurrentStatus = convertToSnakeCase<IRuleStatusSOAttributes>(
+    failures[0].attributes
+  );
   return {
     ...acc,
     [id]: {
       current_status: convertedCurrentStatus,
-      failures: failures.map((errorItem) =>
-        convertToSnakeCase<IRuleStatusAttributes>(errorItem.attributes)
-      ),
+      failures: failures
+        .slice(1)
```

Comment:
this `slice` was missing from the original PR https://github.com/elastic/kibana/pull/79311/files#diff-739fcf7963ad1f59f927b333a119f1447d5a0fcf7b26944335f0252911972b77L68